src/javascripts/modules/Form.js

```diff
@@ -0,0 +1,34 @@
+import "ajaxchimp";
+import AwesomeTingle from "./AwesomeTingle";
+
+const $ = require("jquery");
+
+export default class Form {
+  constructor(el) {
+    this.el = el;
+
+    this.el.addEventListener("keyup", e => {
+      if (e.target.value !== "") {
+        this.el.classList.add("is-filled");
+      } else {
+        this.el.classList.remove("is-filled");
+      }
+    });
```

Comment:
This bit of code makes your form have keyup event as well. I'm assuming this are for input fields only?

Reply:
Yea only for the input. Removing data-module from input and calling listener on this.el.querySelector("#email") from before should fix it.

Reply:
🌮  +1